bullet_train-themes-light/app/views/themes/light/conversations/_message.html.erb

```diff
@@ -126,7 +126,7 @@
             <path d="M0 57.7c.1 145.1 4.1 224 14.5 281.1 19.4 106.3 65.2 156.4 166.3 181.7 48.7 12.1 111.4 18.8 211.7 22.5 23.8.9 196.6 2.8 268.2 3H683l.2 67.8.3 67.8 170.3-102.1L1024 477.3l-170.2-102-170.3-102-.3 67.8-.2 67.9h-75.8c-184.7 0-266.9-4.8-327.2-19-34.2-8.1-62.8-21.1-81.2-37-36.1-31.3-52.3-76.8-58.8-165.5-2-28-3-69.2-3-127.8V0H0v57.7z"/>
           </svg>
           <div class="flex flex-col mr-10 min-w-max">
-            <div class="pt-0 mr-2 ml-4 text-xxs"><%= reply.membership.label_string %> reply: </div>
```

Comment:
Any reason to the colon outside of the locale? One thing that comes to mind is full-width text in Japanese where "：" is different from what we have here, ":". I'd rather be safe and include this in the locale.

Reply:
The intent was to make it reusable. I changed the key to themes.light.conversations.message.reply so it is view-specific and included the ":"

Reply:
Got it!